modules/pxp-module-puppet

```diff
@@ -38,7 +47,7 @@ def last_run_result(exitcode)
 end
 
 def check_config_print(cli_arg, config)
-  command = "#{config["puppet_bin"]} agent --configprint #{cli_arg}"
+  command = "#{quote(config["puppet_bin"])} agent --configprint #{cli_arg}"
```

Comment:
Yeah, thanks for pointing that out.  I was digging in a bit on this PR yesterday as it didn't quite look right... and then when I next looked it got merged.